errbot/backends/telegram_messenger.py

```diff
@@ -369,6 +416,12 @@ def _telegram_special_message(self, chat_id, content, msg_type, **kwargs):
                                              latitude=kwargs.pop('latitude', ''),
                                              longitude=kwargs.pop('longitude', ''),
                                              **kwargs)
+        elif msg_type == 'contact':
```

Comment:
Is it linked to file transfers ? if not please split it in another pull request so we can handle the 2 changes in dependently.